ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php

```diff
@@ -0,0 +1,188 @@
+              'sortable' => TRUE,
+            ],
+            [
+              'size' => 'btn-xs',
+              'links' => [
+                [
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'path' => 'civicrm/admin/pdfFormats?action=update&id=[id]&reset=1',
+                  'task' => '',
+                  'condition' => [],
+                ],
+                [
+                  'entity' => 'OptionValue',
+                  'action' => 'delete',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-trash',
+                  'text' => E::ts('Delete'),
+                  'style' => 'danger',
+                  'path' => '',
+                  'task' => '',
+                  'condition' => [],
+                ],
+              ],
+              'type' => 'buttons',
+              'alignment' => 'text-right',
+            ],
+            [
+              'text' => '',
+              'style' => 'default',
+              'size' => 'btn-xs',
+              'icon' => 'fa-bars',
+              'links' => [
+                [
+                  'entity' => 'OptionValue',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-toggle-on',
+                  'text' => E::ts('Enable Option Values'),
+                  'style' => 'default',
+                  'path' => '',
+                  'task' => 'enable',
+                  'condition' => [],
+                ],
+                [
+                  'task' => 'disable',
+                  'entity' => 'OptionValue',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-toggle-off',
+                  'text' => E::ts('Disable Option Values'),
+                  'style' => 'warning',
+                  'path' => '',
+                  'action' => '',
+                  'condition' => [],
+                ],
+              ],
+              'type' => 'menu',
+              'alignment' => 'text-right',
+            ],
+          ],
+          'actions' => FALSE,
+          'classes' => [
+            'table',
+            'table-striped',
+          ],
+          'draggable' => 'weight',
+          'toolbar' => [
+            [
+              'path' => 'civicrm/admin/pdfFormats?action=add&reset=1',
+              'icon' => 'fa-external-link',
+              'text' => E::ts('Add PDF Page Format'),
+              'style' => 'default',
+              'condition' => [],
+              'task' => '',
+              'entity' => '',
+              'action' => '',
+              'join' => '',
+              'target' => '',
```

Comment:
Typically we create a new item in a popup:

```suggestion
              'target' => 'crm-popup',
```